pkgs/applications/misc/calibre/default.nix

```
@@ -51,6 +90,11 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://raw.githubusercontent.com/debian-calibre/calibre/debian/${finalAttrs.version}+ds-1/debian/patches/hardening/0007-Hardening-Qt-code.patch";
       hash = "sha256-a6yyG0RUsQJBBNxeJsTtQSBV2lxdzz1hnTob88O+SKg=";
     })
+    (fetchpatch {
+      name = "build-from-git-without-internet";
+      url = "https://github.com/kovidgoyal/calibre/commit/4b7c79f71821e9d13edff5d04249c5decf1093a0.patch";
```

IMO it's quite bad for auditability to list this patch file as inheriting from https://github.com/kovidgoyal/calibre instead of from your fork, since you're relying on github making any commit available from any fork here and as a result it looks superficially to someone who doesn't double check, that the commit was actually upstreamed (which it wasn't).

```suggestion
      url = "https://github.com/wrvsrx/calibre/commit/4b7c79f71821e9d13edff5d04249c5decf1093a0.patch";
```